test/simplifyExpression/multiplyFractionsSearch/multiplyFractionsSearch.test.js

```diff
@@ -8,9 +8,13 @@ function testMultiplyFractionsSearch(exprString, outputStr) {
 
 describe('multiplyFractions', function () {
```

Comment:
Hmm, maybe we could merge them. I don't see any other use case for cancelling code besides multiplying fractions.